Assets/MixedRealityToolkit/Internal/Definitions/ButtonDefinition.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace MixedRealityToolkit.Internal.Definitions
+{
+    /// <summary>
+    /// A ButtonDefinition maps the capabilities of a selected controllers buttons, one definition should exist for each button profile.
+    /// </summary>
+    public struct ButtonDefinition
```

Comment:
Should we go ahead and add constructors and make these properties get only?

Reply:
Let's leave constructors off for now and plan to add them later once we see how this is running